mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss

```diff
@@ -1,18 +1,114 @@
 @import "src/styles/variables";
 
+$maxWidth: 1200px;
+
 .pageWrapper {
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 0 $gapMedium;
   margin-top: $marginMedium;
+  gap: $gapBig;
+}
+
+.title {
+  color: var(--secondaryTextButtonColor);
+  text-align: center;
+}
+
+.subtitle {
+  color: var(--primaryTextColor);
+  text-align: center;
+  margin-bottom: $marginSmall;
+}
+
+.aboutBlock {
+  max-width: $maxWidth;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: $gapSmall;
+  margin-top: $marginBig;
+
+  color: var(--secondaryTextColor);
+  background-color: var(primarysBackgroundColor);
+}
+
+.aboutDescription {
+  display: flex;
+  align-items: center;
+  margin-top: $marginBig;
+  background-color: var(--primaryBackgroundColor);
+}
+
+.aboutTextBlock {
+  flex: 1 1 70%;
+  padding: $paddingMedium;
+  display: flex;
+  flex-direction: column;
```

Comment:
we have specific components VerticalContainer for such purposes, use them

Reply:
fixed